pulsar-client-cpp/wireshark/pulsarDissector.cc

```diff
@@ -1025,7 +1033,7 @@ extern __attribute__((unused)) WS_DLL_PUBLIC_DEF const int plugin_want_minor = V
 WS_DLL_PUBLIC void plugin_register(void);
 
 __attribute__((unused)) static void
-proto_reg_handoff_pulsar(void)
+proto_reg_handoff_pulsar()
```

Comment:
There is something special for the `void` argument in C++. See https://softwareengineering.stackexchange.com/questions/286490/what-is-the-difference-between-function-and-functionvoid.

Though this change doesn't affect, you can see the definition of `proto_plugin` here: https://www.wireshark.org/docs/wsar_html/structproto__plugin.html

The function pointer's type is

```c++
void(* register_handoff )(void)
```

So please just keep the code not changed.

Reply:
Thanks for your remind, Addressed.